Client/index.html

```diff
@@ -0,0 +1,150 @@
+                      </div>
+                    </div>
+
+                    <div class="hint clue-hint-container">
+                      <div class="row">
+                        <div class="col-xs-2">
+                          Hint 2
+                        </div>
+                        <div class="col-xs-10">
+                          Begins at the intersection of Second and King Streets near AT&T Park, and travels north, passing under the San Francisco – Oakland Bay Bridge. Continues north past the Ferry Building at Market Street, Fisherman's Wharf, and Pier 39, before ending at Pier 45.
+                        </div>
+                      </div>
+                    </div>
+
+                    <div class="hint clue-hint-container">
+                      <div class="row">
+                        <div class="col-xs-2">
+                          Default Hint
+                        </div>
+                        <div class="col-xs-10">
+                          The pin is located in <a href=""> this radius</a> on the map. 
+                        </div>
+                      </div>
+                    </div>
+
+                    <button class="btn btn-default">Add Hint</button>                
+                  </div>
+                </div>
+              </div>
+              <div class="panel panel-default">
+                <div class="panel-heading" role="tab" id="headingTwo">
+                  <h4 class="panel-title">
+                    <a class="collapsed" data-toggle="collapse" data-parent="#accordion" href="#pin-2" aria-expanded="false" aria-controls="pin-2">
+                      Pin 2: Golden Gate Park
+                    </a>
+                  </h4>
+                </div>
+                <div id="pin-2" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingTwo">
+                  <div class="panel-body">
+                    Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid. 3 wolf moon officia aute, non cupidatat skateboard dolor brunch. Food truck quinoa nesciunt laborum eiusmod. Brunch 3 wolf moon tempor, sunt aliqua put a bird on it squid single-origin coffee nulla assumenda shoreditch et. Nihil anim keffiyeh helvetica, craft beer labore wes anderson cred nesciunt sapiente ea proident. Ad vegan excepteur butcher vice lomo. Leggings occaecat craft beer farm-to-table, raw denim aesthetic synth nesciunt you probably haven't heard of them accusamus labore sustainable VHS.
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+
+        </div>            
+
+      </div>
+    </div>
+
+
+    <!-- <div class="app-container">
+        <div ui-view="create"></div>
+    </div> -->
+
+    <footer class="footer">
+      <!-- footer will go here -->
+    </footer>
+
+  </body>
+  <!-- vendor scripts -->
+  <script src="../bower_components/angular/angular.js"></script>
```

Comment:
See my note below on bower.json. These are no longer listed as bower dependencies and likely need to be removed.